src/node/Main.hs

```diff
@@ -88,8 +88,10 @@ action peerHolder args@Args {..} transport = do
 
     let vssSK = fromJust $ npUserSecret currentParams ^. usVss
     let gtParams = gtSscParams args vssSK
+    -- TODO: this code is impossible to maintain, hence we have 'identity' here.
+    -- Fortunately it's refactored in master.
     let wDhtWorkers :: WorkMode ssc m => KademliaDHTInstance -> ([WorkerSpec m], OutSpecs)
-        wDhtWorkers = (\(ws, outs) -> (map (fst . recoveryCommGuard . (, outs)) ws, outs)) . -- TODO simplify
+        wDhtWorkers = (\(ws, outs) -> (map (fst . identity . (, outs)) ws, outs)) . -- TODO simplify
```

Comment:
This code is removed in master, I don't want to fuck with it in rc branch (comment says it).